codex-rs/exec/src/event_processor.rs

```diff
@@ -186,18 +195,47 @@ impl EventProcessor {
             EventMsg::TokenCount(TokenUsage { total_tokens, .. }) => {
                 ts_println!(self, "tokens used: {total_tokens}");
             }
-            EventMsg::AgentMessageDelta(AgentMessageDeltaEvent { delta: _ }) => {
-                // TODO: think how we want to support this in the CLI
+            EventMsg::AgentMessageDelta(AgentMessageDeltaEvent { delta }) => {
+                if !self.answer_started {
+                    ts_println!(self, "{}\n", "codex".style(self.italic).style(self.magenta));
+                    self.answer_started = true;
+                }
+                print!("{delta}");
+                if let Err(e) = std::io::stdout().flush() {
+                    eprintln!("Failed to flush stdout: {e}");
+                }
```

Comment:
Honestly, maybe we should just do:

```suggestion
                std::io::stdout().flush().expect("could not flush stdout");
```